packages/manager/modules/billing/src/payment/method/add/translations/Messages_fr_FR.json

```diff
@@ -27,6 +27,7 @@
   "billing_payment_method_add_vantiv_recoverable_credit_card_error": "Une erreur est survenue lors de l'ajout de votre moyen de paiement. Veuillez vérifier le numéro de votre carte bancaire puis le ressaisir.",
   "billing_payment_method_add_vantiv_recoverable_card_validation_number_error": "Une erreur est survenue lors de l'ajout de votre moyen de paiement. Veuillez vérifier le numéro de validation de votre carte bancaire puis le ressaisir.",
   "billing_payment_method_add_success": "Votre nouveau moyen de paiement a bien été ajouté.",
+  "billing_payment_method_add_sepa_success": "Vous avez ajouté un compte bancaire en attente de signature électronique (pensez à vérifier votre boite mail), ou en cours de traitement par nos services.",
```

Comment:
For me this sentance seem's weird.
Probably to rewrite by content

Reply:
I just have a confirmation.
yes the sentence is not wrote correctly. 

We have to change it.
Do you know if someone can do it ?

Reply:
Yes I can check with Frederic.

Reply:
```suggestion
  "billing_payment_method_add_sepa_success": "Votre demande d'ajout d'un compte bancaire SEPA a été prise en compte. La demande est en attente d'une signature électronique (pensez à vérifier votre boite mail), ou en cours de traitement par nos services.",
```